pages/content/amp-dev/documentation/guides-and-tutorials/develop/story_ads_best_practices.md

```diff
@@ -260,6 +261,22 @@ There are various tools you can use to encode videos and adjust the quality of t
 
 Ensure the size of your HLS segments are typically no more than 10 seconds in duration.
 
+## Animation
+If you wish to include animation in your story, there are a few things to be aware of. The concept of what is "visible" in an amp-story can be more complicated than other formats.
```

Comment:
Another nit, I'd combine the first two sentences to:

Animations have a few caveats in stories, such as the concept of what is "visible".

Reply:
Done.